### ISSUE TITLE
HLE: cellSysutilCheckCallback aborted 

### ISSUE THREAD

**Issue by raven02** [RPCS3/rpcs3#2022]
It happened when testing Braid which cellSysutilCheckCallback aborted and throw the following error.

> F {PPU[0x7] Thread (main_thread) [0x0032ed8c]} class std::system_error thrown: device or resource busy: device or resource busy

```
s32 cellSysutilCheckCallback(ppu_thread& ppu)
{
	cellSysutil.trace("cellSysutilCheckCallback()");

	const auto cbm = fxm::get_always<sysutil_cb_manager>();

	while (true)
	{
		std::lock_guard<std::mutex> lock(cbm->mutex);

		if (cbm->registered.empty())
		{
			break;
		}

		const auto func = std::move(cbm->registered.front());

		cbm->registered.pop();

		if (s32 res = func(ppu))
		{
			return res;
		}
	}

	return CELL_OK;
}
```

**Comment by raven02** [comment]
This happen since the PPUThread refactoring

**Comment by Nekotekina** [comment]
Ok, it has to unlock the mutex before calling `func(ppu)`

**Comment by raven02** [comment]
Fixed by #2027
